exercises/connect/example.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * "Player O" plays from top to bottom, "Player X" plays from left to right.
+ * @param board
+ */
+export default class {
+  constructor(board) {
+    this.board = board.map(b => b.split(""));
```

Comment:
You can use the (spread)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator] operator here in place of `split`.